bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableParser.scala

```diff
@@ -63,7 +63,7 @@ case object UCAC4Filter extends MagnitudesFilter {
   // UCAC4 ignores A-mags
   override def ignoredMagnitudeField(v: FieldId) = v.id === "amag" || v.id === "e_amag"
   // Magnitudes with value 20 and error over 0.9 are invalid
```

Comment:
Just noted that, as always, documentation is outdated, it should be 'or' rather than 'and'